package/MDAnalysis/coordinates/timestep.pyx

```diff
@@ -0,0 +1,1030 @@
+   .. autoattribute:: has_velocities
+   .. autoattribute:: has_forces
+   .. attribute:: _pos
+
+      :class:`numpy.ndarray` of dtype :class:`~numpy.float32` of shape
+      (*n_atoms*, 3) and internal C order, holding the raw
+      cartesian coordinates (in MDAnalysis units, i.e. Å).
+
+      .. Note::
+
+         Normally one does not directly access :attr:`_pos` but uses
+         the :meth:`~MDAnalysis.core.groups.AtomGroup.coordinates`
+         method of an :class:`~MDAnalysis.core.groups.AtomGroup` but
+         sometimes it can be faster to directly use the raw
+         coordinates. Any changes to this array are immediately
+         reflected in atom positions. If the frame is written to a new
+         trajectory then the coordinates are changed. If a new
+         trajectory frame is loaded, then *all* contents of
+         :attr:`_pos` are overwritten.
+
+   .. attribute:: _velocities
+
+      :class:`numpy.ndarray` of dtype :class:`~numpy.float32`. of shape
+      (*n_atoms*, 3), holding the raw velocities (in MDAnalysis
+      units, i.e. typically Å/ps).
+
+      .. Note::
+
+         Normally velocities are accessed through the
+         :attr:`velocities` or the
+         :meth:`~MDAnalysis.core.groups.AtomGroup.velocities`
+         method of an :class:`~MDAnalysis.core.groups.AtomGroup`
+
+         :attr:`~Timestep._velocities` only exists if the :attr:`has_velocities`
+         flag is True
+
+      .. versionadded:: 0.7.5
+
+   .. attribute:: _forces
+
+      :class:`numpy.ndarray` of dtype :class:`~numpy.float32`. of shape
+      (*n_atoms*, 3), holding the forces
+
+      :attr:`~Timestep._forces` only exists if :attr:`has_forces`
+      is True
+
+      .. versionadded:: 0.11.0
+         Added as optional to :class:`Timestep`
+  
+   .. autoattribute:: dtype
+   .. autoattribute:: dimensions
+   .. autoattribute:: triclinic_dimensions
+   .. autoattribute:: volume
+   .. attribute:: data
+
+      :class:`dict` that holds arbitrary per Timestep data
+
+      .. versionadded:: 0.11.0
+
+   .. automethod:: __getitem__
+   .. automethod:: __eq__
+   .. automethod:: __iter__
+   .. automethod:: copy
+   .. automethod:: copy_slice
+
+"""
+
+
+cdef class Timestep:
+    """Timestep data for one frame
+
+    :Methods:
+
+      ``ts = Timestep(n_atoms)``
+
+         create a timestep object with space for n_atoms
+
+
+    .. versionchanged:: 0.11.0
+       Added :meth:`from_timestep` and :meth:`from_coordinates` constructor
+       methods.
+       :class:`Timestep` init now only accepts integer creation.
+       :attr:`n_atoms` now a read only property.
+       :attr:`frame` now 0-based instead of 1-based.
+       Attributes `status` and `step` removed.
+    .. versionchanged:: 2.0.0
+       Timestep now can be (un)pickled. Weakref for Reader
+       will be dropped.
+       Timestep now stores in to numpy array memory in 'C' order rather than
+       'F' (Fortran).
+    .. versionchanged:: 2.2.0
```

Comment:
I'd make this 2.3.0 (and everywhere else) at the very least (I'm releasing 2.2.0 as soon as possible).

Reply:
Done